docs/ported-functions.md

```diff
@@ -26,14 +27,13 @@ You can view porting progress for the next release
 - [ ] `rhumbDestination`
 - [x] [`rhumbDistance`](./api/turf/org.maplibre.spatialk.turf/rhumb-distance.html)
 - [ ] `square`
-- [x] [`greatCircle`](./api/turf/org.maplibre.spatialk.turf/great-circle.html)
 
 ## Coordinate Mutation
 
 - [ ] `cleanCoords`
 - [ ] `flip`
 - [ ] `rewind`
-- [x] `round` Use `round` or `Math.round` from the standard library instead.
+- [ ] `round`
```

Comment:
unchecking this box because the stdlib round functions don't have a precision parameter, but turf does